keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,40 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  Yes, but CNIs relying on the new field wont recognize it anymore. Also, 
+  because this is a Feature Gate'd API field, disabling the feature gate and 
+  updating a Network Policy with existing EndPort field will remove this field.
 
 * **What happens if we reenable the feature if it was previously rolled back?**
   Nothing. Just need to check if the data is persisted in `etcd` after the 
-  feature is disabled and reenabled or if the data is missed
+  feature is disabled and reenabled. If the object "NetworkPolicy" is updated
+  during this time window between disabling and re-enabling the EndPort field 
+  from this object might be removed by PrepareForUpdate function.
 
 * **Are there any tests for feature enablement/disablement?**
  
- TBD
+ Yes, and the tests are already being implemented in the REST stragegy package
+ for NetworkPolicy.
```

Comment:
Can you point me to them - I think I've never seen enablement/disablement unit tests (we generally test the feature enabled and disablement, but not really what happens when enabling/disabling the feature).

[If we have them that's awesome - I'm looking for examples we can start pointing people to to duplicate the logic]

Reply:
Oh, now I get by the context :)

By https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R92 we test if the feature is enabled or disabled, but not disabling and then re-enabling BUT if this is something desired, I can try to work on a specific test suite only disabling and re-enabling the feature gate without any problem.

Like SetFeatureGate to true, create an object, expect the field, then disable the FeatureGate, get the object, check what fields are present, update the object, check if the field is removed.

Sounds a good approach?

Reply:
Yeah - that sounds fine to me (obviously this doesn't block approving this PR - just mention that in the PRR.